sandbox/libs/authn/src/main/java/org/opensearch/authn/Subject.java

```diff
@@ -29,4 +29,15 @@ public interface Subject {
      * throws SubjectDisabled
      */
     void login(final AuthenticationToken token);
+
+    /**
+     * Logs this subject out and kills any session associated with it
+     */
+    void logout();
```

Comment:
I don't see this method being used outside the tests, why add it?

Reply:
i added it to get PR feedback on whether we would need it. Initially I used it to test multi-login attempts but later just decided to keep it and get PR feedback

Reply:
Can we remove this from the test fix? I don't think logout is something that needs support as there are no persistent sessions with our models only the lifetime of the request

Reply:
okay sounds good. @cwperks @scrawfor99 Let us know if you guys feel otherwise for this.

Reply:
This sounds fine to me.